bittensor/config.py

```diff
@@ -62,7 +63,7 @@ class Config(object):
         Crypto.generate_private_ed25519().public_key())
     try:
         __remote_ip_default__ = requests.get('https://api.ipify.org').text
-    except:
+    except RemoteIPException as e:
```

Comment:
This occurs when the synapse is not internet connected. Allows for local training. Perhaps we should have a log statement here to inform the user? @parall4x @shibshib

Reply:
Added log, and raised exception if requests.get fails.